src/components/manage/Toolbar/More.jsx

```diff
@@ -11,11 +11,11 @@ import { compose } from 'redux';
 import { Link } from 'react-router-dom';
 import { find } from 'lodash';
 
-import { Icon, Display, Workflow } from '../../../components';
-import { getBaseUrl } from '../../../helpers';
+import { Icon, Display, Workflow } from '@plone/volto/components';
+import { getBaseUrl } from '@plone/volto/helpers';
 
-import rightArrowSVG from '../../../icons/right-key.svg';
-import userSVG from '../../../icons/user.svg';
+import rightArrowSVG from '@plone/volto/icons/right-key.svg';
```

Comment:
Unrelated, right? However, leave it! I'm +1 to do a sweep over all relative paths everywhere (we already did that in the lazy load) but it's a good addition before 4 final and will ease the merge for that branch.

Reply:
Yes, it's unrelated, but I figured I'd start to change the ones I encounter on myh way in this polishing phase. :D